junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/DefaultParallelExecutionConfigurationStrategy.java

```diff
@@ -13,7 +13,7 @@
 import static org.apiguardian.api.API.Status.EXPERIMENTAL;
 
 import java.math.BigDecimal;
-
+import java.util.Locale;
```

Comment:
It looks like the deletion of that blank line will fail the build. 

Be sure to run `./gradlew check` before pushing commits.

If there are errors with Spotless, perform the action it suggests to fix formatting.